### PR TITLE
Fixes win10build v2

### PR DESCRIPTION
**Pull request by sab24**
Sorry for all the mess, now this is updated with the three improvements mentioned in #15

### CODE REVIEW

**Comment by sab24** [comment]
```
r2r -L db/extras
Running from /home/sab24/Downloads/r2ghidra/test
Skipping /home/sab24/Downloads/r2ghidra/test/db/extras/r2ghidra because of R2R_SKIP_ARCHOS == 1.
Skipping /home/sab24/Downloads/r2ghidra/test/db/extras/asm_ghidra because of R2R_SKIP_ARCHOS == 1.
Skipping /home/sab24/Downloads/r2ghidra/test/db/extras/ghidra because of R2R_SKIP_ARCHOS == 1.
Skipping /home/sab24/Downloads/r2ghidra/test/db/extras/anal_ghidra because of R2R_SKIP_ARCHOS == 1.
Loaded 0 tests.
Skipping json tests because jq is not available.

Finished in 0 seconds.
```

I suggest upgrading to Archlinux :)

File: cmake/FindRadare2.cmake [hunk 1]
```diff
@@ -26,7 +26,7 @@ if(WIN32)
         find_path(SDB_INCLUDE_DIR
                         NAMES sdb.h sdbht.h sdb_version.h
                         HINTS
-                        ../radare2/include/libr/sdb
+						../radare2/include/libr/sdb
```
Comment (trufae) [inline cmake/FindRadare2.cmake:29]: Wrong indent

File: cmake/FindRadare2.cmake [hunk 2]
```diff
@@ -58,23 +58,24 @@ if(WIN32)
 			magic
 			crypto)
 
-	set(Radare2_LIBRARIES "")
+	set (Radare2_LIBRARIES "" )
```
Comment (trufae) [inline cmake/FindRadare2.cmake:61]: Dont add a space here. Follow the style of the rest of the file

**Review by trufae — CHANGES_REQUESTED** [review]
I think ci.yml only runs in master. Can you change this to test your pr? See how its done in r2cutter or r2 for example

thank you!!

**Comment by sab24** [comment]
I found in DEVELOPERS: "A tab is considered as 8 chars." so this is the right punctuation for the cmake file right?

**Comment by trufae** [comment]
i will probably change that DEVELOPERS thing to follow the r2 coding style, seems a bit inconsistent otherwise to me.

**Comment by trufae** [comment]
What do you mean to use archlinux? the ARCHOS tests are disabled because they require specific stuff from your system to work. and right now this is only suposed to run in ubuntu, i disabled thembecause TCC built binaries contain different sections and maps when debugged and broke some tests. actually ubuntu have LTS which is better for testing, i love archlinux, but using it in the CI may result in unexpected brokenness

**Pull request merged**